js/chirp.js

```diff
@@ -1,7 +1,11 @@
-function chirp(n){
-  // FIX ME
+function chirp(n) {
+  if (n === 0) {
+    return "";
+  }
+  let str = "chirp ";
```

Comment:
Although the test passes, can you figure out a way so that the final string doesn't have an extra space at the end?